packages/commons/src/envUtils.ts

```diff
@@ -301,6 +301,15 @@ const getXRayTraceIdFromEnv = (): string | undefined => {
   return xRayTraceData?.Root;
 };
 
+/**
+ * Helper function to determine if a value is considered thruthy.
+ *
+ * @param value The value to check for truthiness.
+ */
+const isValueTrue = (value: string): boolean => {
```

Comment:
I don't think we need to add this one, we can use the existing `getBooleanFromEnv` above with `extendedParsing` set to `true`